### PR TITLE
about-bulk-upload

### PR DESCRIPTION
**Pull request by g-m-taylor**
Add a page about the bulk upload 

#### Tests
Does this PR include tests for the changes introduced?
- [ ] Yes
- [ ] No, I found it difficult to test
- [x] No, they are not required for this change

### CODE REVIEW

File: src/app/core/breadcrumb/journey.bulk-upload.ts [hunk 1]
```diff
@@ -2,13 +2,21 @@ import { JourneyRoute } from './breadcrumb.model';
...
 export const bulkUploadJourney: JourneyRoute = {
   children: [
     {
       title: 'Bulk upload',
       path: Path.BULK_UPLOAD,
+      children: [
+        {
+          title: 'About bulk upload',
+          path: Path.ABOUT_BULK_UPLOAD,
+          children: [],
```
Comment (chadtomkiss-madetech) [inline src/app/core/breadcrumb/journey.bulk-upload.ts:17]: Don't think we need `children` here

File: src/app/features/bulk-upload/v2/about-bulk-upload/about-bulk-upload.component.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { JourneyType } from '@core/breadcrumb/breadcrumb.model';
+import { BreadcrumbService } from '@core/services/breadcrumb.service';
+
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-about-bulk-upload',
+  templateUrl: './about-bulk-upload.component.html',
+  providers: [],
```
Comment (chadtomkiss-madetech) [inline src/app/features/bulk-upload/v2/about-bulk-upload/about-bulk-upload.component.ts:10]: Do we need `providers` if it's an empty array?

File: src/app/features/bulk-upload/v2/about-bulk-upload/about-bulk-upload.component.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    <p>You always need to upload a staff file with a workplace file, but the training file is optional.</p>
+    <p class="govuk-inset-text">
+      You can download codes and guidance for each file type you need to update (staff, workplace and training).
+    </p>
+    <p>
+      After selecting the files you want to upload, use ASC-WDS to validate them. You'll be able to view any errors and
+      warnings in an error report.
+    </p>
+    <p>A bulk upload is only complete once all errors have been fixed.</p>
+    <button type="submit" class="govuk-button govuk-!-margin-top-6" (click)="return()">Return to bulk upload</button>
```
Comment (chadtomkiss-madetech) [inline src/app/features/bulk-upload/v2/about-bulk-upload/about-bulk-upload.component.html:39]: This button isn't submitting a form and is actually a link.

The markup to make links look like buttons is:

```
<a role="button" draggable="false" class="govuk-button" [routerLink]="['/login']">
    Go to sign in
</a>
```

**Review by chadtomkiss-madetech — CHANGES_REQUESTED** [review]

File: src/app/features/bulk-upload/v2/about-bulk-upload/about-bulk-upload.component.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
```
Comment (chadtomkiss-madetech) [inline src/app/features/bulk-upload/v2/about-bulk-upload/about-bulk-upload.component.ts:1]: Unused `OnDestroy`

**Review by madeviv — CHANGES_REQUESTED** [review]
I've changed the file structure for new BU to be under `/bulk-upload-v2` rather than `/bulk-upload/v2` to make it easier to navigate so you may want to wait for my ticket to be merged first as we might conflict

**Review by madeviv — APPROVED** [review]

**Review by chadtomkiss-madetech — APPROVED** [review]

**Pull request merged**